docusaurus/docs/getting-started.md

```diff
@@ -66,6 +66,44 @@ _`yarn create` is available in Yarn 0.25+_
 
 Follow our [Adding TypeScript](adding-typescript.md) documentation to create a TypeScript app.
 
+## Selecting a package manager
+
+When you run
+
+```sh
+npx create-react-app my-app
+```
+
+It by defaults uses `yarn` to install dependencies, but if you want to opt-in for
+`npm` then just append `--use-npm` to the script above like this;
+
+```sh
+npx create-react-app my-app --use-npm
+```
+
+## Passing through a scripts package
```

Comment:
This should be `###` (heading 3).
 
Would you mind creating a separate PR for the `scripts-version` part? I think it's going to need more discussion, and I'd like to get your first PR in ASAP ;)